context/src/main/java/io/micronaut/scheduling/executor/ExecutorType.java

```diff
@@ -42,5 +42,10 @@ public enum ExecutorType {
     /**
      * @see java.util.concurrent.Executors#newWorkStealingPool()
      */
-    WORK_STEALING
+    WORK_STEALING,
+
+    /**
+     * @see java.util.concurrent.Executors#newVirtualThreadPerTaskExecutor()
+     */
+    THREAD_PER_TASK
```

Comment:
I wonder if we should use another name, like `VIRTUAL_THREAD_PER_TASK` since technically speaking we could implement `THREAD_PER_TASK` with platform threads.

Reply:
it just uses newThreadPerTaskExecutor, so if you don't also configure `virtual: true`, it does use platform threads. so the name is accurate.